packages/bot-web-ui/src/stores/load-modal-store.ts

```diff
@@ -297,6 +317,16 @@ export default class LoadModalStore implements ILoadModalStore {
             return;
         }
 
+        const {
+            dashboard: { active_tab },
+        } = this.root_store;
+
+        if (active_tab === 1 || this.tab_name !== tabs_title.TAB_LOCAL) {
+            if (this.recent_workspace) {
```

Comment:
why not adding the inner if to the main one? 
if ((active_tab === 1 || this.tab_name !== tabs_title.TAB_LOCAL ) && this.recent_workspace)

Reply:
done ✅